prow/git/git.go

```diff
@@ -109,6 +110,11 @@ func (c *Client) getCredentials() (string, string) {
 	return c.user, string(c.tokenGenerator())
 }
 
+// GetBase returns Client.base as a *url.URL
```

Comment:
Why is this necessary? Why do we want a private field if we can just grab the pointer to it?